raiden_contracts/tests/fixtures/channel_test_values_random.py

```diff
@@ -0,0 +1,45 @@
+from random import randint
+from sys import maxsize
+from collections import namedtuple
+
+
+ChannelValues = namedtuple('ChannelValues', [
+    'deposit',
+    'withdrawn',
+    'transferred',
+    'locked'
+])
+
+
+def channel_settle_test_values_random(n):
+    return [
+        (
+            tuple([
+                ChannelValues(
+                    deposit=deposit,
+                    withdrawn=withdrawn,
+                    transferred=transferred,
+                    locked=locked
+                )
+                for deposit, withdrawn, transferred, locked in
+                [sorted(randint(0, maxsize) for _ in range(4)) for _ in range(n)]
+            ]),
+            True
+        ),
+        (
+            tuple([
+                ChannelValues(
+                    deposit=deposit,
+                    withdrawn=withdrawn,
+                    transferred=transferred,
+                    locked=locked
+                )
+                for withdrawn, transferred, deposit, locked in
+                [sorted(randint(0, maxsize) for _ in range(4)) for _ in range(n)]
```

Comment:
As mentioned, it is hard to understand what you are trying to achieve here. Either use readable code or make sure to comment. The two cases look similar and you have to pay attention to see that the `withdrawn, transferred, deposit, locked` values are actually reordered (though I don't really understand why).

`[sorted(randint(0, maxsize) for _ in range(4)) for _ in range(n)]` seems like you are choosing sets of 4 sorted random values. In the `True` case, you will always have `deposit <= withdrawn <= transferred <= locked`, which is not really useful and not correct. Am I wrong?